### ISSUE TITLE
[REQ] Bot logs over hosted website possible?

### ISSUE THREAD

**Issue by DineshValor** [masterking32/MasterHamsterKombatBot#309]
I was thinking, is it possible to show logs on website (live) ?

Like [here](https://hamstervalor.onrender.com/) hosted the bot and made a web page, so easily visiting there to check bot running logs!

**Comment by qawechi** [comment]
Yup it is possible. There are many free servers u can use. I have a server and i run the bot there. It is 24/7 running without stopping.

**Comment by copycatcodex** [comment]
how about if there are updates with the bot?
